src/App.tsx

```diff
@@ -20,12 +22,19 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={searchWord}
+                onChange={(event) => {
+                  setSearchWord(event.target.value);
+                }}
               />
             </div>
           </div>
         </div>
 
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList
+          movies={moviesFromServer}
+          searchWord={searchWord}
+        />
```

Comment:
No need to use filter logic inside MovieList component. Also your task says you need to create state variable with `visibleMovies` and pass it here as props.